templates/core/terraform/airlock/service-bus.tf

```diff
@@ -0,0 +1,71 @@
+resource "azurerm_servicebus_namespace" "airlock_sb" {
```

Comment:
Can we use the existing sb? These are expensive.

Reply:
Fixed